lib/cisco_node_utils/upgrade.rb

```diff
@@ -58,7 +58,15 @@ def self.image_version(image=nil, uri=nil)
       if image && uri
         config_get('upgrade', 'image_version', image: image, uri: uri)
       else
-        config_get('show_version', 'version').split(' ')[0]
+        version = config_get('show_version', 'version')
+        # show version displays version differently for release and
+        # development builds.
+        # Eg: release build
+        #       NXOS: version 7.0(3)I4(2)
+        # Eg: development build
+        #       NXOS: version 7.0(3)IFD6(1) [build 7.0(3)IGD7(0.65)]
+        return version.split(' ')[0] unless version[/build\s+\S+]/]
```

Comment:
Why do we need to split for the release build case?  Can't we just return version unchanged?